functions.go

```diff
@@ -10,6 +13,34 @@ type Function struct {
 	Return     Type
 }
 
+// IsImplementedForSyscall returns a boolean value indicating whether the function is implemented for syscall or not.
```

Comment:
"returns a boolean value indicating" can be simplified to "reports".

"Foobar reports whether ..." is an idiomatic way of documenting Go functions that return `bool`.

Reply:
Done.